### ISSUE TITLE
bsmart key error

### ISSUE THREAD

**Issue by rictech** [FelixFrog/pdfgrabber#72]
```
Traceback (most recent call last):
  File "C:\Disco_D\000D Documenti\APPOGGIO PROGRAMMI\PDFGrabber_Launcher-main\main.py", line 271, in <module>
    main()
  File "C:\Disco_D\000D Documenti\APPOGGIO PROGRAMMI\PDFGrabber_Launcher-main\main.py", line 253, in main
    downloadbook()
  File "C:\Disco_D\000D Documenti\APPOGGIO PROGRAMMI\PDFGrabber_Launcher-main\main.py", line 174, in downloadbook
    pdfpath = utils.downloadbook(service, token, bookid, books[bookid], progressfun)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Disco_D\000D Documenti\APPOGGIO PROGRAMMI\PDFGrabber_Launcher-main\utils.py", line 42, in downloadbook
    pdf = service.downloadbook(token, bookid, data, progress)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Disco_D\000D Documenti\APPOGGIO PROGRAMMI\PDFGrabber_Launcher-main\services\bsm.py", line 110, in downloadbook
    pid, label = resmd5[md5]
                 ~~~~~~^^^^^
KeyError: 'a6cd47f06060b659756537f19df4c756'
```

**Comment by FelixFrog** [comment]
Should be fixed in [ad4eb38](https://github.com/FelixFrog/pdfgrabber/commit/ad4eb380f19dc2cfc1f260d06a8daac6209a63eb)

**Comment by rictech** [comment]
solved, thank you
